recipe/meta.yaml

```diff
@@ -7,16 +7,10 @@ package:
 source:
   - url: https://github.com/lief-project/LIEF/archive/{{ version }}.tar.gz
     fn: lief-{{ version }}.tar.gz
-    sha256: 8f1a12d69d9f6bd8404d4f5809549d1c7f3138cacf3bccd1192cb4f46bfb9eb7
-    patches:
-      - 0001-Allow-building-from-released-tarball.patch
-      - 0002-MSVC-Add-bigobj-flag.patch
-      - 0003-Export-LIEF-Object-and-LIEF-MachO-DataCodeEntry-clas.patch
-      - 0004-MachO-Fix-has_dynamic_symbol_command.patch
-      - 0005-Link-Python-extension-module-to-shared-LIEF-library.patch
```

Comment:
All patches apart from the last one have been included upstream.
The last one is outdated due to https://github.com/lief-project/LIEF/pull/201.